_posts/2019-11-27-release-1.8.3.md

```diff
@@ -0,0 +1,155 @@
+---
+layout: post
+title:  "Apache Flink 1.8.3 Released"
+date:   2019-11-27 12:00:00
+categories: news
+authors:
+- hequn:
+  name: "Hequn Cheng"
+---
+
+The Apache Flink community released the third bugfix version of the Apache Flink 1.8 series.
+
+This release includes 43 fixes and minor improvements for Flink 1.8.2. The list below includes a detailed list of all fixes and improvements.
+
+We highly recommend all users to upgrade to Flink 1.8.3.
+
+Updated Maven dependencies:
+
+```xml
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-java</artifactId>
+  <version>1.8.3</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-streaming-java_2.11</artifactId>
+  <version>1.8.3</version>
+</dependency>
+<dependency>
+  <groupId>org.apache.flink</groupId>
+  <artifactId>flink-clients_2.11</artifactId>
+  <version>1.8.3</version>
+</dependency>
+```
+
+You can find the binaries on the updated [Downloads page]({{ site.baseurl }}/downloads.html).
+
+List of resolved issues:
+
+<h2>        Sub-task
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-13723'>FLINK-13723</a>] -         Use liquid-c for faster doc generation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-13724'>FLINK-13724</a>] -         Remove unnecessary whitespace from the docs&#39; sidenav
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-13725'>FLINK-13725</a>] -         Use sassc for faster doc generation
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-13726'>FLINK-13726</a>] -         Build docs with jekyll 4.0.0.pre.beta1
+</li>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-13791'>FLINK-13791</a>] -         Speed up sidenav by using group_by
+</li>
+</ul>
+        
+<h2>        Bug
+</h2>
+<ul>
+<li>[<a href='https://issues.apache.org/jira/browse/FLINK-12342'>FLINK-12342</a>] -         Yarn Resource Manager Acquires Too Many Containers
```

Comment:
I went over the release notes and updated a few JIRA titles and categories; so this needs to be updated beyond FLINK-14995

Reply:
Thanks a lot for the update and fixing the license problem. I will update this PR once FLINK-14995 is addressed.